build/bin/user_setup

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+set -x
+
+# ensure $HOME exists and is accessible by group 0 (we don't know what the runtime UID will be)
+mkdir -p ${HOME}
+chown ${USER_UID}:0 ${HOME}
+chmod ug+rwx ${HOME}
+
+# runtime user will need to be able to self-insert in /etc/passwd
+chmod g+rw /etc/passwd
+
+# no need for this script to remain in the image after running
```

Comment:
This seems counter to how a script should be run: propose removing it in the caller instead of self-removing.

Reply:
this is from the operator, just re-using the same thing here

Reply:
I confess I haven't stayed up to date with the operator -- can you elaborate on what it's doing that makes that required here?